tests/java-client/build.gradle.kts

```diff
@@ -0,0 +1,57 @@
+import com.apollographql.apollo3.annotations.ApolloExperimental
+
+plugins {
+  id("java")
+  id("com.apollographql.apollo3").version("4.0.0-beta.7")
```

Comment:
Gradle noob question: is there a way to not have to set the version here, but use what's in `build-logic`?

Reply:
Add `com.apollographql.apollo3:apollo-gradle-plugin` to your build-logic implementation and remove the version from here